docs: Add prerequisite for typescript configuration (#539)
faker-js/faker: README.md

### TypeScript Support

> In order to have faker working properly, you need to check that in your `tsconfig` file those compilerOptions are set correctly:

Reviewer: ```suggestion
In order to have faker working properly, you need to check if these `compilerOptions` are set correctly in your `tsconfig` file:
```
